electron-builder.json

```diff
@@ -77,7 +77,13 @@
       }
     ],
     "hardenedRuntime": true,
-    "gatekeeperAssess": true
+    "gatekeeperAssess": true,
+    "entitlements": "./build/entitlements.mac.plist",
+    "entitlementsInherit": "./build/entitlements.mac.plist",
+    "extendInfo": {
+      "NSMicrophoneUsageDescription": "Third party plugins (like jitsi) will use the microphone to provide in-app videoconference",
+      "NSCameraUsageDescription": "Third party plugins (like jitsi) will use the microphone to provide in-app videoconference"
```

Comment:
Potentially, plugins can use camera and microphone for many things, not just videoconferencing. We might want to give a more vague approach to this. 0/5

Reply:
Definitely there can be a better wording, I'll wait for @esethna comment on it.

Reply:
@Willyfrog do you have a screenshot of how this prompt would appear?

Reply:
I don't! I didn't take one when it asked and now it doesn't allow me to get it to ask again.

It is a standard OS modal, I don't really have access to it. I can share a build with you so you can test if you prefer